charts/kueue/templates/kueue-viz/cluster-role-binding.yaml

```
@@ -0,0 +1,14 @@
+{{- if .Values.enableKueueViz }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kueue-backend-read-access-binding
+  namespace: system
```

```suggestion
  namespace: '{{ .Release.Namespace }}'
```
I think it should be something like this.

---

correct